yt/frontends/gamer/data_structures.py

```diff
@@ -179,14 +179,21 @@ def _validate_parent_children_relationship(self):
             # edges between children and parent
             for c in grid.Children:
                 for d in range(0, 3):
-                    assert grid.LeftEdge[d] <= c.LeftEdge[d], (
-                        "Grid %d, Children %d, Grid->EdgeL %14.7e, Children->EdgeL %14.7e"
+                    msgL = (
```

Comment:
I don't know how often this code is called, but it'd probably be better to move `msgL` and `msgR` into the if statements, so that they are lazy-evaluated.

Reply:
yeah I was aiming for that originally but I think I left them out of the if block to save 4 chars and avoid splitting the strings so as to make them a little more readable. I think the readability gain is probably worth this minor a perf hit but I'm open to diverging opinions.

Reply:
Fine.